src/FirstSteps/IHeroesDamage.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FirstSteps
+{
+    public interface IHeroesDamage
```

Comment:
The name is not quite general. Maybe something like IAttacker would be better and will fit both hero and boss. Or maybe you could invent something better.